whylog/tests/tests_client/test_files/003_match_time_range/parsers.yaml

```diff
@@ -1,10 +1,12 @@
 ---
 name: cause
-regex: ^(\d+) root cause
-params:
-    - {type: time, format: '%s'}
+regex_str: ^(\d+) root cause
```

Comment:
This parser does not match with the line content: "2015-12-03 12:08:08 root cause"